src/CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ if(ENABLE_COVERAGE)
 	target_link_libraries(${VC4C_LIBRARY_NAME} gcov "-fprofile-arcs -ftest-coverage")
 	target_link_libraries(${VC4C_PROGRAM_NAME} gcov "-fprofile-arcs -ftest-coverage")
 endif(ENABLE_COVERAGE)
+if(OPT_FOUND)
+	target_compile_definitions(${VC4C_LIBRARY_NAME} PRIVATE OPT_PATH="${OPT_FOUND}")
```

Comment:
This is not necessary for the first version but might be added later: For builds with LLVM library, we might be able to use the optimizer library instead of calling the prgram.